components/atoms/figure-component.html

```diff
@@ -0,0 +1,59 @@
+<div class="component" title="figure Component" version="1.0">
+<style>
+.figure {
+    display: grid;
+    margin: 0 auto;
+}
+
+.figure img {
+    width: 100%;
+    margin-bottom: 1em;
+}
+
+.figure figcaption {
+    text-align: center;
+}
+.figure figcaption a{
+    text-decoration: none;
+}
+</style>
+
+
+  <figure class="figure">
+      <img src="../../img/mustafa.jpg" alt="">
```

Comment:
Use placeholder image instead of real image. You can do this instead:

<img src="https://via.placeholder.com/150">